modules/nf-core/ashlar/main.nf

```diff
@@ -0,0 +1,33 @@
+process ASHLAR {
+    tag '$meta.id'
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::ashlar=1.17.0" : null)
+    container 'docker.io/labsyspharm/ashlar:1.17.0'
+
+    input:
+    tuple val(meta), val(file_in)
+
+    output:
+    tuple val(meta), path("*.tif")  ,   emit: ashlar_tif
+    path "versions.yml"                 ,   emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args_conf = task.ext.args ?: ''
+    def args_meta = meta.args ?: ''
```

Comment:
I just realized putting args in meta doesn't make sense, since the sample sheet will be one row per cycle, and the args need to be per ashlar call i.e. per experiment (many cycles).

Reply:
Don't the meta args apply to the ashlar call?
In the existing (non-sample sheet) code in main.nf (right after the commented code above) we have:
```
    ashlar \\
        $file_in \\
        $args_conf \\
        $args_meta
```
Then in the sample sheet version (in the ashlar_sheet branch) we have additional args for each row.  So maybe we don't need the additional args in the sample sheet?
Am I misunderstanding this?